content/docs/11-security.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: "Security"
+metaTitle: "Spectro Cloud Palette"
+metaDescription: "Spectro Cloud provides scalable, policy-based cluster management of Kubernetes for enterprises that need a high degree of control over their infrastructure, whether it is in public cloud, private cloud, bare metal or any combination."
+icon: "lock" 
+hideToC: true
+fullWidth: true
+hiddenFromNav: false
+isIntegration: false
+hideMenuSidebar: false 
+hideToCSidebar: false
+---
+
+import SecurityWP from "shared/components/common/SecurityWP"; 
+ 
+
+<SecurityWP
```

Comment:
The layout feels awkward and frankly unnecessary.  All we need to do in this page is to provide a high-level overview and provide a link to each topic. The current layout will be challenging to migrate when we switch over to a documentation framework.